include/seqan3/alignment/configuration/align_config_mode.hpp

```diff
@@ -48,14 +58,26 @@ inline constexpr detail::global_alignment_type global_alignment;
 namespace seqan3::align_cfg
 {
 
+/*!\brief Helper variable to select the local alignment.
+ * \ingroup alignment_configuration
+ *
+ * \details
+ *
+ * ### Example
+ *
+ * \snippet snippet/alignment/configuration/align_cfg_mode_example.cpp local
+ */
+inline constexpr detail::local_alignment_type local_alignment;
```

Comment:
please put this into the seqan3 namespace above and change on all places.

Reply:
Okay!